glib/src/convert.rs

```diff
@@ -317,6 +317,64 @@ pub fn locale_to_utf8(opsysstring: &[u8]) -> Result<(crate::GString, usize), Cvt
     }
 }
 
+#[doc(alias = "g_utf8_to_ucs4")]
+pub fn utf8_to_utf32(str: &[u8]) -> Result<(Slice<u32>, usize), CvtError> {
```

Comment:
I initially wanted to make this return a `Slice<u32>` instead of `Slice<char>`, but I'm not sure whether the layouts of u32 and char are guaranteed to be the same, so that casting the returned pointer from `*mut u32` to `*mut char` would be sound, any thoughts?

Reply:
I would've expected this to be `&str -> Slice<char>` :)